### ISSUE TITLE
The result return from stop() does not have `duration` value

### ISSUE THREAD

**Issue by iNima** [shadow-app/flutter_audio_recorder#8]
Hi there,

When I stop the audio recording I expect to receive the total duration of the recorded audio in the `duration` parameter of the result.
`var result = await this.audioRecorder.stop();`
But the `duration` of the result is always 0.

Currently, a workaround for this would be this:
```
Recording recording = await this.audioRecorder.current();
Duration duration = recording.duration;
var result = await this.audioRecorder.stop();
```
Which is probably not that accurate since it is a little bit before actually stopping the recorder.

Could you provide the correct duration in the result of stop() method as well?

**Comment by nikli2009** [comment]
Hi buddy, just released `0.5.3`, in this build, it's able to use `stop` to get last status of recording
```
var result = await this.audioRecorder.stop();
```

Would you like to have a try ? :)

**Comment by iNima** [comment]
Great! It works correctly now.
Thanks @nikli2009
